### ISSUE TITLE
Migrate Triplet Network from PyTorch to TensorFlow

### ISSUE THREAD

**Issue by VecherVhatuX** [VecherVhatuX/learning_data_science_in_prison#1172]
Migrate Triplet Network from PyTorch to TensorFlow, replace PyTorch-specific modules with TensorFlow counterparts and update loss function and model training to be compatible with TensorFlow, to provide an alternative implementation of the Triplet Network.
